src/DI/DbalExtension.php

```diff
@@ -79,8 +80,14 @@ class DbalExtension extends CompilerExtension
 	/** @var AbstractPass[] */
 	protected array $passes = [];
 
-	public function __construct()
+	public function __construct(
+		private ?bool $debugMode = null
```

Comment:
Please, update documentation and add at least one test for auto-detection and one test for providing debugMode over constructor.

Reply:
Ok, I have edited one of the existing tests and added another for testing disabled debug mode, so we have all cases covered, last test is doing it with debug.panel.